common/openapi.yml

```diff
@@ -241,25 +241,6 @@ paths:
         500:
           $ref: '#/components/responses/ServerError'
 
-  /api/pipelineruns/v1/pipelineruns:
```

Comment:
Since this is an API removal I am thinking this will be a major version change. What are your thoughts?

Reply:
makes sense to me